coap-gateway/coapconv/grpcErr2CoapCode.go

```diff
@@ -60,3 +61,11 @@ func GrpcCode2CoapCode(statusCode codes.Code, operation Operation) coapCodes.Cod
 	}
 	return coapCodes.InternalServerError
 }
+
+func GrpcErr2CoapCode(err error, operation Operation) coapCodes.Code {
```

Comment:
the `operation` parameter is never used in the body of the function

Reply:
Fixed